### PR TITLE
Jsinha/migrate all tests tobk

### PR DESCRIPTION
**Pull request by jaymalasinha**
### Description

Migrate all travis verification tests to buildkite

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Comment by jaymalasinha** [comment]
@markan is working on refactoring the oc_erchef and bookshelf verification tests which currently fail in travis too.  This PR is ready for merge after review.

File: src/oc_bifrost/Makefile [hunk 1]
```diff
@@ -16,6 +16,13 @@ endif
 
 # If the user hasn't set POSTGRES_USER, default to postgres
 POSTGRES_USER ?= 'postgres'
+POSTGRES_CMD = sudo -u  postgres psql -U $(POSTGRES_USER)
```
Comment (markan) [inline src/oc_bifrost/Makefile]: Let's revert all of the changes here, except for the travis: ci lines below.

In other words, let's make the only change to this file parallel to the changes you made in the other makefiles.

File: .expeditor/verify.pipeline.yml [hunk 2]
```diff
@@ -35,5 +41,164 @@ steps:
...
+    - cd /workdir/omnibus; make install
+    - make ci
+  expeditor:
+    executor:
+      docker:
+        privileged: true
+        image: "chefes/a1-buildkite"
+        environment:
+          - LUALIB=~/.luarocks/lib/lua/5.2
+          - ELIXIR_VERSION=1.4
```
Comment (marcparadise) [inline .expeditor/verify.pipeline.yml]: Does Chef Server ship with any elixir components?

**Comment by markan** [comment]
I think this is merge-ready. Please rebase and squash before merging to keep the history pretty.

**Comment by jaymalasinha** [comment]
@markan @marcparadise @tduffield Thank you for your review and feedback.  I was able to fix the license_scout tests too and squashed the commits.  This is ready for merge now.  Can one of you please force merge this PR for me.  @markan  will fix the oc-erchef and bookshelf tests in a separate PR.  Thanks again.

File: .expeditor/license_scout.sh [hunk 3]
```diff
@@ -5,6 +5,10 @@ set -eou pipefail
 if [[ "${EXPEDITOR:-false}" == "true" ]]; then
   apt-get update
   apt-get install -y libpq-dev libsqlite3-dev
+  # Pin ruby to 2.5.3 since chef-server tests heavily depend
+  asdf global ruby 2.5.3
```
Comment (tduffield) [inline .expeditor/license_scout.sh]: Any reason not to use `local` instead of `global`?

File: .expeditor/license_scout.sh [hunk 4]
```diff
@@ -39,4 +39,4 @@ for dir in "${erlang_install_dirs[@]}"; do
 done
 
 echo "+++ Running License Scout"
-license_scout --only-show-failures
+  license_scout --only-show-failures
```
Comment (tduffield) [inline .expeditor/license_scout.sh]: unnecessary indentation?

File: scripts/bk_tests/bk_install.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+apt-get --purge remove -y postgresql-9.3
+apt-get update -y && apt-get install -y lua5.1 luarocks postgresql-9.6
+cp /workdir/scripts/bk_tests/pb_hba.conf /etc/postgresql/9.6/main/pg_hba.conf
+asdf global erlang 18.3
+asdf global ruby 2.5.3
+gem install bundler --version '1.17.3' --no-document
```
Comment (tduffield) [inline scripts/bk_tests/bk_install.sh]: Is there a reason we're installing 1.17.3 specifically? Why not `~> 1.17` ?

File: .expeditor/license_scout.sh (hunk 3 above)
Comment (jaymalasinha) [reply .expeditor/license_scout.sh]: local does not update the default tools_version and current license_scout gem (2.1.5) does not support ruby 2.6.1

**Pull request merged**